draft-kazuho-httpbis-priority.md

```diff
@@ -384,6 +372,23 @@ set to `4` and the incremental parameter set to `1`.
 priority = u=4, i=?1
 ~~~
 
+## Defining New Parameters
+
+When attempting to extend priorities, care must be taken to ensure any use of
+existing parameters are either unchanged or modified in a way that is backwards
+compatible for peers that are unaware of the extended meaning.
+
+# The Priority HTTP Header Field
+
+The Priority HTTP header field can appear in requests and responses. A client
+uses it to specify the priority of the response. A server uses it to inform
+the client that the priority was overwritten. An intermediary can use the
+Priority information from client requests and server responses to correct or
+amend the precedence to suit it (see {{merging}}).
+
+The Priority header field is an end-to-end signal of the request
```

Comment:
Should we specifically say that the intermediary should not change the priority and if it wants to apply a different priority on the next hop, it should use a frame?  I feel like we get confused when we start talking about intermediaries modifying the priority header in various ways.